apps/website/src/content/projects.json

```diff
@@ -0,0 +1,159 @@
+[
+  {
+    "name": "clr.fund",
+    "description": "a protocol for efficiently allocating funds to public goods that benefit the Ethereum Network.",
+    "hackathon": null,
+    "status": "Production",
+    "links": {
+      "website": "https://clr.fund/",
+      "github": "https://github.com/clrfund/monorepo/",
+      "discord": "https://discord.com/invite/ZnsYPV6dCv"
+    }
+  },
+  {
+    "name": "MACI_QF in Allo",
+    "description": "a MACI Quadratic Funding strategy to be used within Gitcoin's Allo Protocol.",
+    "hackathon": null,
+    "status": "Production",
+    "links": {
+      "github": "https://github.com/gitcoinco/MACI_QF"
+    }
+  },
+  {
+    "name": "MACI-RPGF",
+    "description": "a fork from Easy-RPGF that enables any community to easily run an RPGF round using MACI.",
```

Comment:
can we change this to: a platform that allows to run different types of vote and funding rounds using MACI.